talkative-tangs/Bangazon-LLC-2
# OE-Product-Page

## Link to Ticket
[Ticket 13 - Needs Quantity Missing](https://github.com/talkative-tangs/Bangazon-LLC-2/issues/13)

## Description of Proposed Changes
-Added Individual Product Category Templaes/URLs/Views

## Steps to Test

Go to the product category page.
Click on each product category.
Ensure the URL matches the category you're seeing.
Ensure the items in the category are correctly displayed.

```sh
git fetch --all
git checkout oe-product-page

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Templates
* Views
* URLs

## Mentions @username

@CurtainUp 
@BryanNilsen
@laboyd001
